olp-cpp-sdk-authentication/src/AuthenticationClient.cpp

```diff
@@ -591,19 +591,13 @@ client::CancellationToken AuthenticationClient::Impl::HandleUserRequest(
           return;
         }
 
-        switch (response_status) {
-          case http::HttpStatusCode::OK: {
-            // Cache the response
-            cache->locked(
-                [credentials,
-                 &response](utils::LruCache<std::string, SignInUserResult>& c) {
-                  return c.InsertOrAssign(credentials.GetKey(), response);
-                });
-          }
-          default:
-            callback(response);
-            break;
+        if (response_status == http::HttpStatusCode::OK) {
+          cache->locked([credentials, &response](
+                            utils::LruCache<std::string, SignInUserResult>& c) {
+            return c.InsertOrAssign(credentials.GetKey(), response);
+          });
         }
+        callback(response);
```

Comment:
This change is unrelated to the commit

Reply:
It was needed to avoid error on Linux Clang build.

Reply:
Could you please submit it as a separate PR? So in the case, we need to revert anything, the fix stays in the master

Reply:
Sorry, I mean Gcc build fixes, will fix them in separate PR.